Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1

```diff
@@ -175,7 +179,16 @@ function Set-ClusterProtectionLeveltoEncryptandSignforServiceFabric
 
         [String]
         [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies the path to the file to be used as input for the remediation")]
-        $FilePath
+        $FilePath,
+
+        [String]	
```

Comment:
Add the parameter details : Take reference from the app service PR.

Reply:
This is resolved, thanks